### ISSUE TITLE
In text-to-lexeme, show sense link

### ISSUE THREAD

**Issue by Ainali** [fnielsen/ordia#155]
For a sense, it is very useful to quickly know if it already is connected to other concepts or lexemes. Primarily, these properties are of interest (and usually, only one of them are present on each sense):

* [item for this sense (P5137)](https://www.wikidata.org/wiki/Property:P5137)
* [predicate for (P9970)](https://www.wikidata.org/wiki/Property:P9970)
* [demonym of (P6271)](https://www.wikidata.org/wiki/Property:P6271)
* [pertainym of (P8471)](https://www.wikidata.org/wiki/Property:P8471)

If they are lacking, it might also be interesting to know if any of these properties are present (but perhaps that should be covered by another issue):

* [translation (P5972)](https://www.wikidata.org/wiki/Property:P5972)
* [synonym (P5973)](https://www.wikidata.org/wiki/Property:P5973)
* [antonym (P5974)](https://www.wikidata.org/wiki/Property:P5974)

**Comment by dpriskorn** [comment]
Is the purpose of this to analyze a text and see if all words found make sense somehow in the graph?

**Comment by Ainali** [comment]
Yes, my purpose is to paste short Wikipedia articles to see if they are theoretically possible to generate with lexemes. That is, to see if there is enough information about the lexemes that generators would be able to find them. For that, a concept link is important, but, for example, a sense image is not.

**Comment by dpriskorn** [comment]
So a new column with a title like this Has [item for this sense (P5137)](https://www.wikidata.org/wiki/Property:P5137)
    [predicate for (P9970)](https://www.wikidata.org/wiki/Property:P9970)
    [demonym of (P6271)](https://www.wikidata.org/wiki/Property:P6271)
    [pertainym of (P8471)](https://www.wikidata.org/wiki/Property:P8471) with the value true/false for each word would be enough for your purposes?

**Comment by dpriskorn** [comment]
Perhaps a column for each of those properties is more valuable to you so you see the values also?

**Comment by Ainali** [comment]
A column each would, of course, be the most clear thing.

However, since it is very unlikely that there is more than one of these as statements, I could also envision just one column titled "concept linked with" containing as values the name of the property linked to that statement or a blank if there was nothing. (This could save a lot of space.)
